src/views/Pool.vue

```diff
@@ -2124,6 +2350,11 @@ export default {
         }
       );
 
+      const status = await result.wait();
+      if (status) {
+        await this.checkAllbalances();
+      }
```

Comment:
will be better to create a wrapper for transaction result instead of make changes at every place where the tx sends

Reply:
Done